linkinator.config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "recurse": true,
+  "skip": [
+    "https://codecov.io/gh/googleapis/",
+    "www.googleapis.com",
```

Comment:
Let's add a follow issue so we don't forget to clear these out

Reply:
https://github.com/googleapis/nodejs-secret-manager/issues/5